CONTRIBUTING.md

```diff
@@ -99,12 +109,22 @@ The workflow is as follows for a **new** protocol:
         2.  alternatively, download the rendered version: see [these instructions](REVIEWING.md)
 
     5.  add text, media, ... to the Rmarkdown files
-
+    
     6.  save your changes
 
     7.  stage, commit, push changes
 
-8.  When finished, go to your draft pull request and press 'ready for review' and add reviewers.
+8.  When finished, go to your draft pull request and press 'ready for review' 
+
+   1. Wait for the continuous integration checks to finish and see if you the checks succeeded.
+   These checks will run `protocolhelper::check_frontmatter()` and `protocolhelper::check_structure()`.
+   In case there are problems with the YAML front matter of your `index.Rmd` file or problems with the structure of the protocol not confirming with the protocol templates, these problems will be listen and can be consulted.
+   
+       1.  Address the problems detected by `protocolhelper::check_frontmatter` or `protocolhelper::check_structure`. You can see the list of problems by clicking on the check online (at the bottom of the pull request webpage). Alternatively, you can run these functions locally and see the list of problems printed in your console.
```

Comment:
```suggestion
       1.  Address the problems detected by `protocolhelper::check_frontmatter` or `protocolhelper::check_structure`. You can see the list of problems by clicking on the check online (at the bottom of the pull request webpage). Alternatively, you can run these functions locally and see the list of problems printed in your console.
        Note that during this automatic run, a commit to update the version number may be automatically pushed to the remote.
        This should not pose a problem for local clones as this change can be automatically included and will not cause conflicts (i.e. `git pull origin main` with a local protocol branch checked out)
```

Reply:
This should not pose a problem for local clones as this change can be automatically included and will not cause conflicts (i.e. \`git pull origin main\` with a local protocol branch checked out)

@ElsLommelen I don't understand what you mean with the last sentence. Anyway it may be too scary for contributors and perhaps unneeded, so it can be dropped?

Reply:
I copied it more or less from a phrase of @hansvancalster in RELEASE.md, and it refers to the fact that the commit that is automatically generated in the GHA, should be pulled locally.  But you are right, it should be written less complicated.  What about something like `If this commit is added, press pull in the RStudio Git Pane to add it to your local clone.` to replace the last sentence?

As git often gives an error and asks to pull remote changes first before pushing additional commits, I would in any case write something, as this error may scare starting Github users as well.  (Or they may just click the message away without noticing their commit is not pushed to the origin.)

Reply:
Your suggestion is OK indeed!

Now, it seems I was confused because the check-functions, either run in GHA or locally (line 122), cause a new commit to be 'pushed to the remote' (quoting your line 123 above). If that happened from the local repo, then pulling makes no sense. Maybe it doesn't happen that way when run locally, but that's not entirely clear here.

Reply:
No, this new commit is pushed from the GHA to the remote by also running `update_version_number()` additionally to the check-functions.  So a local run of the check-functions will not cause a new commit (running `update_version_number()` locally will, but no need anymore to bother contributors with that and mention it in the guidelines).

Reply:
Thanks for clarifying. Your suggested change (quoted below) is already sufficient to prevent further confusion, so fine to add that change!

> If this commit is added, press pull in the RStudio Git Pane to add it to your local clone.

Reply:
edited above.